<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="23320" started-at="2018-08-22T05:14:57Z" finished-at="2018-08-22T05:15:21Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="23320" started-at="2018-08-22T05:14:57Z" finished-at="2018-08-22T05:15:21Z">
      <class name="com.qa.sweety.tests.NewLoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.sweety.tests.NewLoginPageTest@28ac3dc3]" name="setUp" is-config="true" duration-ms="4447" started-at="2018-08-22T10:44:57Z" finished-at="2018-08-22T10:45:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="validLoginTest()[pri:1, instance:com.qa.sweety.tests.NewLoginPageTest@28ac3dc3]" name="validLoginTest" duration-ms="4800" started-at="2018-08-22T10:45:02Z" finished-at="2018-08-22T10:45:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLoginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.sweety.tests.NewLoginPageTest@28ac3dc3]" name="tearDown" is-config="true" duration-ms="110" started-at="2018-08-22T10:45:07Z" finished-at="2018-08-22T10:45:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.sweety.tests.NewLoginPageTest@28ac3dc3]" name="setUp" is-config="true" duration-ms="3215" started-at="2018-08-22T10:45:07Z" finished-at="2018-08-22T10:45:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="invalidPasswordTest()[pri:2, instance:com.qa.sweety.tests.NewLoginPageTest@28ac3dc3]" name="invalidPasswordTest" duration-ms="3917" started-at="2018-08-22T10:45:10Z" finished-at="2018-08-22T10:45:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Signed in successfully.] but found [Invalid email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Signed in successfully.] but found [Invalid email or password.]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:453)at org.testng.Assert.assertEquals(Assert.java:463)at com.qa.sweety.tests.NewLoginPageTest.invalidPasswordTest(NewLoginPageTest.java:66)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidPasswordTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.sweety.tests.NewLoginPageTest@28ac3dc3]" name="tearDown" is-config="true" duration-ms="155" started-at="2018-08-22T10:45:14Z" finished-at="2018-08-22T10:45:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.sweety.tests.NewLoginPageTest@28ac3dc3]" name="setUp" is-config="true" duration-ms="2989" started-at="2018-08-22T10:45:14Z" finished-at="2018-08-22T10:45:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="invalidEmailTest()[pri:3, instance:com.qa.sweety.tests.NewLoginPageTest@28ac3dc3]" name="invalidEmailTest" duration-ms="3522" started-at="2018-08-22T10:45:17Z" finished-at="2018-08-22T10:45:20Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Signed in successfully.] but found [You need to sign in or sign up before continuing.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Signed in successfully.] but found [You need to sign in or sign up before continuing.]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:453)at org.testng.Assert.assertEquals(Assert.java:463)at com.qa.sweety.tests.NewLoginPageTest.invalidEmailTest(NewLoginPageTest.java:79)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidEmailTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.sweety.tests.NewLoginPageTest@28ac3dc3]" name="tearDown" is-config="true" duration-ms="111" started-at="2018-08-22T10:45:20Z" finished-at="2018-08-22T10:45:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.sweety.tests.NewLoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
